Dockerfile.ros1

```diff
@@ -1,44 +1,31 @@
 ARG ROS_DISTRIBUTION=noetic
-FROM ros:$ROS_DISTRIBUTION-ros-core
+FROM ros:$ROS_DISTRIBUTION-ros-base
 ARG ROS_DISTRIBUTION=noetic
 
-# Install system dependencies
-RUN apt-get update && apt-get install -y --no-install-recommends \
-  python3-pip \
-  python3-rosdep \
-  python3-wstool \
-  build-essential \
-  nlohmann-json3-dev \
-  libasio-dev \
-  libssl-dev \
-  libwebsocketpp-dev \
-  && rm -rf /var/lib/apt/lists/*
-
-# Initialize rosdep
-RUN rosdep init && rosdep update
-
-# Add common files and ROS 1 source code
-COPY CMakeLists.txt /ros1_ws/src/ros-foxglove-bridge/CMakeLists.txt
+# Add package.xml so we can install package dependencies.
 COPY package.xml /ros1_ws/src/ros-foxglove-bridge/package.xml
-COPY foxglove_bridge_base /ros1_ws/src/ros-foxglove-bridge/foxglove_bridge_base
-COPY ros1_foxglove_bridge /ros1_ws/src/ros-foxglove-bridge/ros1_foxglove_bridge
 
 # Install rosdep dependencies
 RUN . /opt/ros/$ROS_DISTRIBUTION/setup.sh && \
-    apt-get update && rosdep install -y \
+    apt-get update && rosdep update && rosdep install -y \
       --from-paths \
         /ros1_ws/src \
       --ignore-src \
     && rm -rf /var/lib/apt/lists/*
 
-# Use the bash shell so we can use the source command
-SHELL [ "/bin/bash" , "-c" ]
+# Add common files and ROS 1 source code
+COPY CMakeLists.txt /ros1_ws/src/ros-foxglove-bridge/CMakeLists.txt
+COPY foxglove_bridge_base /ros1_ws/src/ros-foxglove-bridge/foxglove_bridge_base
+COPY ros1_foxglove_bridge /ros1_ws/src/ros-foxglove-bridge/ros1_foxglove_bridge
 
-# Create a Catkin workspace
-RUN mkdir -p /ros1_ws/src
+# Build and install gtest.
+RUN cd /usr/src/gtest \
+  && cmake CMakeLists.txt \
+  && make \
+  && cp ./*.a /usr/lib || true
```

Comment:
This is required for melodic, I'm not sure why we need this, I believe that it should also work without it.